pkg/scanners/universal/scanner.go

```diff
@@ -40,6 +41,7 @@ func New(opts ...options.ScannerOption) *Scanner {
 			json.NewScanner(opts...),
 			yaml.NewScanner(opts...),
 			toml.NewScanner(opts...),
+			helm.New("mysql", opts...),
```

Comment:
Guessing this should be something else and not "mysql" but I'm not sure what.

Reply:
whoops